csp/pull.csp

```diff
@@ -45,6 +56,13 @@ function execute() {
     }
 }
 
+function updateContext() {
+    const contextSelect = document.getElementById('newContext');
```

Comment:
I'd recommend just using "var" instead of "const" (in case we ever end up using this from Studio)